main.go

```diff
@@ -197,14 +198,64 @@ func runBenchmark(cmdStr string, args []string) (parse.Set, error) {
 		return nil, xerrors.Errorf("failed to run '%s' command: %w", cmd, err)
 	}
 
-	b := bytes.NewBuffer(out)
-	s, err := parse.ParseSet(b)
+	s, err := parseBenchmark(out)
 	if err != nil {
 		return nil, xerrors.Errorf("failed to parse a result of benchmarks: %w", err)
 	}
 	return s, nil
 }
 
+func runPreviousBenchmark(cmdStr string, args []string) (parse.Set, error) {
+	prevSet, err := runBenchmark(cmdStr, args)
+	if err != nil && strings.Contains(err.Error(), "exit status 1") {
```

Comment:
we should probably match any exit status because blocking a pipeline because of any error on previous code is not good

Reply:
I think golang always return exit status 1 for any error during the execution of the test.

Reply:
not sure... even with panics?

Reply:
done